resdk/resources/base.py

```diff
@@ -26,39 +26,43 @@ class BaseResource(object):
 
     endpoint = None
 
+    WRITABLE_FIELDS = ('slug', 'name', 'permissions')
```

Comment:
I have made the lists of fields class constants. How do you like the new solution?